edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Amqp/AmqpProtocolHead.cs

```diff
@@ -116,7 +116,6 @@ public async Task CloseAsync(CancellationToken token)
 
         public void Dispose()
```

Comment:
Can we remove this ?

Reply:
It needs to implement IDisposable because IProtocolHead extends it.